test/extended/openstack/topology.go

```diff
@@ -0,0 +1,92 @@
+package openstack
+
+import (
+
+	//	"github.com/gophercloud/gophercloud/v2"
+
+	"github.com/openshift/openstack-test/test/extended/openstack/client"
+
+	"github.com/gophercloud/gophercloud/v2"
+	"github.com/gophercloud/gophercloud/v2/openstack"
+	vavailabilityzones "github.com/gophercloud/gophercloud/v2/openstack/blockstorage/v3/availabilityzones"
+	cavailabilityzones "github.com/gophercloud/gophercloud/v2/openstack/compute/v2/availabilityzones"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	//"github.com/openshift/openstack-test/test/extended/openstack/client"
+
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	//e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenShift cluster", func() {
+	defer g.GinkgoRecover()
+
+	var dc dynamic.Interface
+	var clientSet *kubernetes.Clientset
+	oc := exutil.NewCLI("openstack")
+
+	//var computeClient *gophercloud.ServiceClient
+
+	g.BeforeEach(func(ctx g.SpecContext) {
+		g.By("preparing a dynamic client")
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err = dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		clientSet, err = e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
+	})
+	g.It("enables topology aware scheduling when compute and volume AZs are identical", func(ctx g.SpecContext) {
```

Comment:
Would you mind to add a reference to the bug that resolved this issue as a comment?